v1/examples/life-events/death-registered.json

```diff
@@ -73,9 +73,7 @@
         "description": "Deceased found on 3 August 1989"
       },
       "freeFormatDeathDate": "Unknown due to mortuary error but some time in July 1989",
-      "deathRegistrationTime": {
-        "value": "1989-08-10T10:22:04"
-      }
+      "deathRegistrationTime": "1989-08-10T10:22:04"
```

Comment:
An aside, but we should figure out what timezone this is when provided by GRO. I'd want to convert to Zulu time I think

Reply:
Yeah we ought to call this out in the RFC, because even if we allow `deathRegistrationTime` to be local time, `toe` is still Zulu so we definitely have to do some conversion.

Reply:
I agree that a date generated by a system should be reported in UTC, this aligns with properties of the JWT like `toe`. I think this helps in incident resolution where we may have to look at different component logs running across different timezones / data centre locations.  If consumers want to translate to their local time zone for display purposes then that can be down to them.

Reply:
Proposal in https://github.com/alphagov/di-identity-vocab/pull/47